README.md

```diff
@@ -8,10 +8,19 @@ This project is in a very early stage and will be both restructured and renamed.
 
 ## Quickstart
 
-to install the python and other developer requirements into a venv run:
+To install the python and other developer requirements into a venv run:
 
     make install
 
+### Configuration options
+
+There's a few env vars that can be used with this project:
+
+* `TEST_TARGET`: Set to `AWS_CLOUD` to use an externally-deployed instance when running tests.
```

Comment:
note: In the future this shouldn't be part of the library. For now it's ok to include it in the README as it's still included :+1: